lib/matplotlib/colors.py

```diff
@@ -1203,61 +1203,67 @@ class DivergingNorm(TwoSlopeNorm):
     ...
 
 
+def _make_norm_from_scale(scale_cls, base_cls=None, *, init=None):
```

Comment:
Can we get a docstring for this? I _think_ `init` exists purely for signature purposes, but that requires reading all the code to figure that out.

Reply:
Wrote up something quick.